source/slides/07_agile_methods.rst

```diff
@@ -0,0 +1,220 @@
+.. _07_agile_methods:
+
+Agile Methodologies
+===================
+
+Agile Methodologies
+-------------------
+
+In general, agile development methodologies involve:
+
+* Iteration of process
+* Short cycles
+* Fast feedback
+* Parallel execution of various stages
+* Less documentation-focused
+* Timeboxing (Limiting time spent)
```

Comment:
Might include some background on why its important to understand these methodologies from a sysadmin point of view. We may have already covered this before, but its a good refresher.